daemon/config/config_linux.go

```diff
@@ -174,6 +178,7 @@ func setPlatformDefaults(cfg *Config) error {
 	cfg.SeccompProfile = SeccompProfileDefault
 	cfg.IpcMode = string(DefaultIpcMode)
 	cfg.Runtimes = make(map[string]types.Runtime)
+	cfg.CDISpecDirs = cdi.DefaultSpecDirs
```

Comment:
As `cdi.DefaultSpecDirs` is a slice, the `cfg.CDISpecDirs` slice will share the same backing array and therefore e.g. e.g. `cfg.CDISpecDirs[0] = "foo"` will also mutate `cdi.DefaultSpecDirs[0]` and any other `cfg.CDISpecDirs` slices which haven't been completely replaced. Better make a copy of the slice for safety.
```suggestion
	cfg.CDISpecDirs = append([]string(nil), cdi.DefaultSpecDirs...)
```

Reply:
Thanks. Will update.